src/Components/NavBar/Menus/UserMenu.tsx

```diff
@@ -30,8 +34,23 @@ export const UserMenu: React.FC = () => {
     })
   }
 
+  const isAdmin = user.type === "Admin"
```

Comment:
There's a [`userIsAdmin` helper](https://github.com/artsy/reaction/blob/master/src/Utils/user.ts#L34) that can be used to make this decision. I know it's only one line, but if some day we change the rules for what makes an admin, it'd be nice to have those decisions all made in one place.

Reply:
Oh yeah, that's much nicer; will update